cmake/projects/SocketIOClient/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2013-2017, Ruslan Baratov, Richard Hodges
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+        PACKAGE_NAME
+        SocketIOClient
+        VERSION
+        "1.6.1"
+        URL
+        "https://github.com/madmongo1/socket.io-client-cpp/archive/v1.6.1-hunter.tar.gz"
```

Comment:
I've made a fork for you:
* https://github.com/hunter-packages/socket.io-client-cpp